packages/arb-token-bridge-ui/src/components/TransactionHistory/TransactionsTable/TransactionsTableDepositRow.tsx

```diff
@@ -218,7 +220,12 @@ export function TransactionsTableDepositRow({
       </td>
 
       <td className="w-1/5 whitespace-nowrap px-3 py-3">
-        {formatAmount(Number(tx.value), { symbol: tx.asset.toUpperCase() })}
+        {formatAmount(Number(tx.value), {
+          symbol: sanitizeTokenSymbol(tx.asset.toUpperCase(), {
```

Comment:
We can move the `toUpperCase()` inside the function

Reply:
Done